src/Semantic/Api/Terms.hs

```diff
@@ -112,14 +114,20 @@ instance ShowTerm Python.Term where
   showTerm = serialize Show . void . Python.getTerm
 
 
-sexprTermParsers :: Map Language (SomeParser SExprTerm Loc)
-sexprTermParsers = aLaCarteParsers
+sexprTermParsers :: PerLanguageModes -> Map Language (SomeParser SExprTerm Loc)
+sexprTermParsers = allParsers
 
 class SExprTerm term where
-  sexprTerm :: (Carrier sig m, Member (Reader Config) sig) => term Loc -> m Builder
+  sexprTerm :: term Loc -> Builder
 
 instance (ConstructorName syntax, Foldable syntax, Functor syntax) => SExprTerm (Term syntax) where
-  sexprTerm = serialize (SExpression ByConstructorName)
+  sexprTerm = SExpr.serializeSExpression ByConstructorName
```

Comment:
I didn’t want to bother trying to push this interface into `Serializing.Format` and ensure it supports diffs, so I’m just calling directly into the serializations.